QEfficient/exporter/export_hf_to_cloud_ai_100.py

```diff
@@ -206,6 +206,10 @@ def convert_to_cloud_kvstyle(
 
     if tokenizer.pad_token_id is None:
         tokenizer.pad_token_id = tokenizer.eos_token_id
+        
+    # If Pad token is out of range of vocab size
+    if tokenizer.pad_token_id >= tokenizer.vocab_size:
+        tokenizer.pad_token_id = tokenizer.vocab_size - 1
```

Comment:
Did the model export and compilation pass through? Outputs verified for model card: `Salesforce/xgen-7b-8k-base`

Reply:
Yes, on local it passed successfully.

Reply:
Let wait for CI to pass and will merge then.